sgf-viewer/src/App.svelte

```diff
@@ -92,7 +92,7 @@
         display: flex;
         justify-content: center;
         padding: 0 min(1.5em, 4vw);
-        width: 100vw;
+        width: 98vw;
```

Comment:
not an issue that's new to this PR but i was noticing a horizontal scrollbar on our website, reducing this fixes it (though maybe there's some other root cause here)